geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und/oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet, auf welche Weise Stimmen gezählt oder geschätzt und ob sie anschließend protokolliert werden.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## § 5 Anträge
+
+(1) Anträge sind schriftlich zu verfassen und innerhalb von 14 Tagen nach Zugang der Einladung zur Vollversammlung bei den einladenden Personen einzureichen.
+
+(2) Die einladenden Personen entscheiden über die Aufnahme des Antrags in die Tagesordnung.
+
+(3) Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge werden vor der Abstimmung des jeweiligen Antrags behandelt.
+
+(2) Änderungsanträge können von der den ursprünglichen Antrag stellenden Person direkt angenommen werden. Geschieht dies nicht, stimmt die Vollversammlung über die Annahme ab.
+
+(3) Änderungsanträge können vor der Vollversammlung schriftlich bekanntgegeben und begründet werden.
+
+## § 7 Anträge zur Geschäftsordnung
+
+(1) Anträge zur Geschäftsordnung werden durch das Heben beider Arme angezeigt. Davon kann abgewichen werden, wenn die Intention für die Versammlungsleitung eindeutig erkennbar ist.
+
+(2) Anträge zur Geschäftsordnung werden unverzüglich behandelt.
+
+(3) Vor der Abstimmung darf je eine antragsberechtigte Person für bzw. gegen den Antrag das Wort erhalten.
+
+(4) Geschäftsordnungsanträge bedürfen einer Zweidrittelmehrheit der anwesenden Mitglieder der Vollversammlung.
+
+(5) Geschäftsordnungsanträge sind:
+  1. Antrag auf Vertagung eines Tagesordnungspunktes (einfache Mehrheit),
+  2. Antrag auf den Schluss der Redeliste,
+  3. Antrag auf den Schluss der Beratung und ggf. sofortige Abstimmung,
+  4. Antrag auf Ende der Sitzung,
+  5. Antrag auf Zählung und/oder Protokollierung des Abstimmungsergebnisses (auf Verlangen einer Person).
+  
+Die hier genannten Quoren gelten abweichend zu § 7 (4).
+
+## § 8 Beschlüsse
+
+(1) Beschlüsse werden grundsätzlich offen mit einfacher Mehrheit der anwesenden Stimmberechtigten gefasst, sofern die Satzung der Fachschaft oder diese Geschäftsordnung nichts anderes vorschreiben. Werden mehr Enthaltungsstimmen als Ja- und Nein-Stimmen zusammen abgegeben, gilt der Antrag als abgelehnt. Stimmengleichheit gilt als Ablehnung.
+
+(2) Abstimmungen erfolgen durch Handzeichen.
+
+(3) Beschlüsse können auch in Form von Meinungsbildern oder Stellungnahmen gestaltet werden.
+
+## § 9 Änderung der Geschäftsordnung
+
+Die Vollversammlung kann Änderungen dieser Geschäftsordnung mit einer Zweidrittelmehrheit aller anwesenden Stimmberechtigten beschließen.
```

Comment:
GO enthält keine Vorschriften über Online-VV. Sollte man vielleicht aufnehmen, da ja z.B. § 7 (1) in einer Online-VV nicht gut realisierbar ist.

Z.B.

> § 9 wird § 10
>
> § 9 wird neu gefasst: Sofern zwingende Belange dies erforderlich machen, kann die Vollversammlung auf digitalem Wege stattfinden. Die Vorschriften gelten entsprechend. Sofern technologische Einschränkungen dies erforderlich machen, kann die Sitzungsleitung nach pflichtgemäßem Ermessen von Verfahrensvorschriften abweichen. Die Abweichung ist unzulässig, wenn mindestens 10 Mitglieder der Vollversammlung dagegen Einspruch erheben.

Reply:
Findest du in der GO gerade Regelungen, die in einer Online-VV kaputt gehen? Falls ja, weise uns gerne darauf hin! Wir haben keine gefunden, weshalb eine gesonderte Betrachtung nicht nötig ist